### PR TITLE
adds required html, css files

### PR DESCRIPTION
**Pull request by edumorlom**
Adds required public files such as index.html.
Adds CSS files.
Removes helper_functions.py, no longer needed.

### CODE REVIEW

File: covid19-dashboard/public/index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=1000">
+    <link rel="icon" href="%PUBLIC_URL%/favicon.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="View The Top States/Counties in the USA Affected By COVID-19"
```
Comment (beets) [inline covid19-dashboard/public/index.html:26]: please update

File: covid19-dashboard/public/index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    />
+    <meta http-equiv="cache-control" content="max-age=0" />
+    <meta http-equiv="cache-control" content="no-cache" />
+    <meta http-equiv="expires" content="0" />
+    <meta http-equiv="expires" content="Tue, 01 Jan 1980 1:00:00 GMT" />
+    <meta http-equiv="pragma" content="no-cache" />
+    <!--    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />-->
```
Comment (beets) [inline covid19-dashboard/public/index.html:33]: please strip out these instructional comments

File: covid19-dashboard/public/index.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      If you open it directly in the browser, you will see an empty page.
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-174481330-1"></script>
```
Comment (beets) [inline covid19-dashboard/public/index.html:64]: please give the team access to this account

File: covid19-dashboard/public/index.html (hunk 2 above)
Comment (edumorlom) [reply covid19-dashboard/public/index.html:33]: done

File: covid19-dashboard/public/index.html (hunk 3 above)
Comment (edumorlom) [reply covid19-dashboard/public/index.html:64]: i removed it for now, I dont think we need it. We should probably use the same account as other tools no?

File: covid19-dashboard/public/index.html (hunk 1 above)
Comment (edumorlom) [reply covid19-dashboard/public/index.html:26]: I don't want to hard code COVID text, so I removed the meta tag.
I will add the tags through JavaScript using the Config.json file.

File: covid19-dashboard/src/index.css [hunk 4]
```diff
@@ -28,17 +45,6 @@
 
 /* HEADER */
 
-.date {
```
Comment (edumorlom) [inline covid19-dashboard/src/index.css]: Date will no longer use this style.

Comment (edumorlom) [reply covid19-dashboard/src/index.css]: Removing style.

**Comment by edumorlom** [comment]
Ready for re-review!

**Comment by edumorlom** [comment]
@beets this PR contains the CSS file that has the 'clickable' class.

**Review by beets — APPROVED** [review]
yikes, sorry for the delay on this!

**Pull request merged**